### ISSUE TITLE
Unhealty Container

### ISSUE THREAD

**Issue by noire-x** [immauss/openvas#171]
Hi There,
i am having some issues with the container, looks like it is looping on
![image](https://user-images.githubusercontent.com/112881201/220435214-15efb6fe-9cc4-451a-af95-e74e74179372.png)
And on portainer it says unhealty
![image](https://user-images.githubusercontent.com/112881201/220435287-199e2049-91b4-420a-810a-12d7193775ba.png)

Do you have any suggestion?
Thanks!

**Comment by immauss** [comment]
Hmmm ... that seems normal. It is supposed to do a health check every 60 seconds. 
Is anything not working?

**Comment by noire-x** [comment]
I launched a scan anyway both docker ps openvas and portainer showing "unhealty", but the scan is running it is at 40% now, will post an update soon ty

**Comment by noire-x** [comment]
Looks like all working even with unhealty container, ty!

**Comment by immauss** [comment]
I've been looking at this for a while now, and I can't see why the health check reports no problems on completion, but the container still reports unhealthy. This might take me a while to figure out . . . . .

**Comment by immauss** [comment]
Good grief......
OK ... so after spending WAY too much time trying to figure out what was wrong with my health check script ....

I went back to the source for HEALTHCHECK.
https://docs.docker.com/engine/reference/run/#healthcheck

Where I found this command:
``` docker inspect --format='{{json .State.Health}}' <container-name> ```

This lead me to the realization that the script was fine, but it was taking more than 3 seconds to complete ... so the HEALTHCHECK was hitting the timeout. 

<sigh>

The script took about 5 seconds to run on my machine, so I raised the timeout in the Dockerfile to 10s. This should give more than enough time to complete the checks even on slower hardware. 

The current latest tag is now operating with this setting and running fine and healthy on my production servers. 

Thanks for opening this. 

-Scott
